src/main/java/com/example/JavaProject/controller/IngredientController.java

```diff
@@ -0,0 +1,33 @@
+package com.example.JavaProject.controller;
+
+import com.example.JavaProject.dto.RecipeDto;
+import com.example.JavaProject.response.IngredientResponse;
+import com.example.JavaProject.response.LikesCountResponse;
+import com.example.JavaProject.response.RecipeResponse;
+import com.example.JavaProject.service.interfaces.AuthenticationService;
+import com.example.JavaProject.service.interfaces.IngredientService;
+import com.example.JavaProject.service.interfaces.RecipeService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+@RequiredArgsConstructor
+public class IngredientController {
```

Comment:
RecipeService and AuthenticationService are not used, you should remove them

and optimise imports for this file